content/1_General/IO.mdx

```diff
@@ -495,6 +520,24 @@ int main() {
 
 Again, `BufferedReader` and `PrintWriter` should be used. The provided constructor `public Kattio(String problemName)` handles file input / output for you.
 
+```java
```

Comment:
Mention Kattio further down? Doesn't really make sense that it says "provided constructor `public Kattio(String problemName)`" here when it's provided much later.

Reply:
Ok i don't understand why include kattio at all, literally no one will use it in cp. they'll just use bufferedreader or scanner

Reply:
it's template that's faster to copy-paste than to write out bufferedreader/printwriter